chore(deps): upgrade dependencies

aws-cdk-automation opened the pull request:
Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/aws/aws-cdk-cli/actions/runs/21342224904

------

*Automatically created by projen via the "upgrade" workflow*

rix0rrr commented:
My guess is this update is blocked on another "AWS SDK has peculiarities on GitHub Actions" problem.

github-actions[bot] committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/aws/aws-cdk-cli/actions/runs/21342224904

------

*Automatically created by projen via the "upgrade" workflow*

Signed-off-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
